src/platform/amd/vangogh/include/arch/xtensa/tie/xt_datacache.h

```diff
@@ -0,0 +1,7 @@
+#ifndef __cstub_xt_datacache_h__
+#define __cstub_xt_datacache_h__
+
+#include "Xm_ACP_5_0_001_PROD.h"
```

Comment:
this file has been removed

Reply:
Had removed the inclusion too.
Thanks @cujomalainey